src/libs/ascent/runtimes/flow_filters/ascent_runtime_blueprint_filters.cpp

```diff
@@ -359,29 +388,70 @@ BlueprintPartition::execute()
 
     conduit::Node *n_output = new conduit::Node();
     
-    conduit::Node n_options = params();
+    conduit::Node n_options;
```

Comment:
we have already validated the input in verify params, we can simply pass the options along as is -- we don't need to pick out the individual params and copy them.